src/main/java/by/triumgroup/recourse/controller/CourseController.java

```diff
@@ -26,4 +29,20 @@
     @GetMapping(value = "search", params = "status")
     List<Course> searchByStatus(@RequestParam("status") Course.Status status, Pageable pageable);
 
+    @PostMapping("{courseId}/registration")
+    @ResponseStatus(HttpStatus.OK)
+    void registerToCourse(@PathVariable("courseId") Integer courseId, @Auth UserAuthDetails authDetails);
+
+    @DeleteMapping("{courseId}/registration")
```

Comment:
I see, it's delete request, but maybe it will be better to point out method functionality in mapping path too.